docs/develop/java/temporal-nexus.mdx

```diff
@@ -0,0 +1,654 @@
+        client.newWorkflowStub(HelloCallerWorkflow.class, workflowOptions);
+    logger.info("Workflow result: {}", helloWorkflow.hello("Nexus", NexusService.Language.ES));
+    logger.info(
+        "Started workflow workflowId: {} runId: {}",
+        WorkflowStub.fromTyped(helloWorkflow).getExecution().getWorkflowId(),
+        WorkflowStub.fromTyped(helloWorkflow).getExecution().getRunId());
+  }
+}
+```
+
+<!--SNIPEND-->
+
+## Make Nexus calls across Namespaces with a development Server {#nexus-calls-across-namespaces-dev-server}
+
+To run the tutorial, follow the steps below to run the Nexus handler Worker, the Nexus caller Worker, and the starter.
+
+### Run Workers connected to a local development server
+
+In separate terminal window, run the Nexus handler Worker:
+
+```bash
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.handler.HandlerWorker \
+    --args="-target-host localhost:7233 -namespace my-target-namespace"
+```
+
+In another terminal window, run the Nexus caller Worker:
+
+```bash
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.caller.CallerWorker \
+    --args="-target-host localhost:7233 -namespace my-caller-namespace"
+```
+
+### Start a caller Workflow
+
+With the Workers running, the final step in the local development process is to start a caller Workflow.
+
+Run the starter:
+
+```bash
+./gradlew -q execute -PmainClass=io.temporal.samples.nexus.caller.CallerStarter \
+    --args="-target-host localhost:7233 -namespace my-caller-namespace"
+```
+
+This will result in:
+
+```
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Workflow result: Nexus Echo 👋 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Started workflow workflowId: 9b3de8ba-28ae-42fb-8087-bdedf4cecd39 runId: 404a2529-764d-4d1d-9de5-8a9475e40fba 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Workflow result: ¡Hola! Nexus 👋 
+[main] INFO  i.t.s.nexus.caller.CallerStarter - Started workflow workflowId: 9cb29897-356a-4714-87b7-aa2f00784a46 runId: 7e71e62a-db50-49da-b081-24b61016a0fc
+```
+
+## Make Nexus calls across Namespaces in Temporal Cloud {#nexus-calls-across-namespaces-temporal-cloud}
+
+This section assumes you are already familiar with [how connect a Worker to Temporal Cloud](https://docs.temporal.io/develop/java/core-application#run-a-temporal-cloud-worker).
+The same [source code](https://github.com/temporalio/samples-go/tree/main/nexus) is used in this section, but the `tcld` CLI will be used to create Namespaces and the Nexus Endpoint, and mTLS client certificates will be used to securely connect the caller and handler works to their respective Temporal Cloud Namespaces.
+
+### Install the latest tcld CLI and generate certificates
+
+To install the latest version of the `tcld` CLI, run the following command:
+
+```
+brew install temporalio/brew/tcld
+```
+
+If you don’t already have certificates, you can generate them for mTLS Worker authentication using the command below:
+
+```
```

Comment:
Maybe mention that they're valid in this example for 1 year?

Reply:
Added a note below https://github.com/temporalio/documentation/pull/3160/commits/e362237937feddb32b1497f8ed2498fcf3277012